File: server/Models/Recipe.cs
using System.ComponentModel.DataAnnotations;

public class Recipe
{
    public int Id { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
    [MaxLength(255)] public string Title { get; set; }
    [MaxLength(5000)] public string Instructions { get; set; }
    [Url, MaxLength(1000)] public string Img { get; set; }
    public RecipeCategory Category { get; set; }
    public string CreatorId { get; set; }
    public Account Creator { get; set; }
}

public enum RecipeCategory
{
    Breakfast,
    Lunch,
    Dinner,
    Snack,
    Dessert
}

File: client/src/models/Recipe.js
export class Recipe{
    constructor(data) {
        this.id = data.id;
        this.title = data.title;
        this.category = data.category;
        this.img = data.img;
        this.instructions = data.instructions;
        this.creatorId = data.creatorId;
        this.favorite = false;
        this.creator = data.creator;
    }
}